lib/dns.js

```diff
@@ -289,31 +289,38 @@ function setServers(servers) {
   // servers cares won't have any servers available for resolution
   const orig = this._handle.getServers();
   const newSet = [];
-  const IPv6RE = /\[(.*)\]/;
   const addrSplitRE = /(^.+?)(?::(\d+))?$/;
 
   servers.forEach((serv) => {
+    // Easily recognized?
```

Comment:
I am not sure what the comment stands for.

Reply:
I would personally prefer a comment that's a descriptive statement. The question mark makes it pretty unclear.